decentraland/unity-renderer
# fix: chat not working

## What does this PR change?

Fixes a problem where chats are not loading correctly.
Removed chat web socket messages from the mappings, so they fallback correctly to the `Bridges` object.

https://user-images.githubusercontent.com/56365551/218132726-a647cd92-5c06-487c-b46b-e9e9eefc3505.mp4


## How to test the changes?

1. Go to: https://play.decentraland.zone/?explorer-branch=fix/chat-not-loading
2. Open the chat list from the chat icon in the taskbar
3. Check that channels are loading
4. Open any channel including ~nearby, you should be able to retrieve the history and send messages
5. Send a DM to a user

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
